.github/workflows/publish-release.yml

```diff
@@ -53,14 +49,14 @@ jobs:
         -   name: Build and Test
             run: make test
 
-        # Deploy to S3
+        # Upload to S3
         -   name: Configure AWS credentials
-            if: github.ref == 'refs/heads/main' # Only deploy on merge into main
             uses: aws-actions/configure-aws-credentials@v1.6.1
             with:
                 aws-region: ${{secrets.AWS_REGION}}
                 role-to-assume: ${{ secrets.AWS_OIDC_RUNNER_ROLE }}
 
-        -   name: Sync to S3
-            if: github.ref == 'refs/heads/main'
-            run: aws s3 sync --quiet ./examples/build/ s3://pyscript.net/alpha/
+#        -   name: Sync to S3
+#            run: | # Overwrite "latest" alpha + versioned subdirectory
+#              aws s3 sync --quiet ./examples/build/ s3://pyscript.net/alpha/${{ github.ref_name }}
+#              aws s3 sync --quiet ./examples/build/ s3://pyscript.net/alpha/
```

Comment:
Why aren't we pushing the changes to S3?

Reply:
Will uncomment after some testing. Just making sure I don't overwrite the Alpha without a backup plan if it doesn't work as intended.